keywords/custom_keywords.resource

```diff
@@ -0,0 +1,31 @@
+*** Settings ***
+Library    String
+Library    DateTime
+Resource    ../browser/browser.resource
+Resource    ../page_objects/register_page.resource
+Resource    ../page_objects/login_page.resource
+Resource    ../page_objects/header_navigation.resource
+Variables    ../test_data/default_account.py
+
+*** Keywords ***
+# Generate Random Email Address
+#     ${random_str}=    Generate Random String    8    [LOWER]
+#     ${random_email}=    Catenate     SEPARATOR=    ${random_str}    @mailinator.com
+#     RETURN    ${random_email}
+
+Generate Random Email Based On DateTime
+    ${current_time}=    Get Current Date
+    ${conver_time}=    Convert Date    ${current_time}    epoch
+    Log To Console    ${conver_time}
+    ${random_email}=    Catenate    SEPARATOR=    ${conver_time}    @mailinator.com
+    RETURN    ${random_email}
```

Comment:
Please define utils/data_helper and put all these methods here

Reply:
I created file /utils/data_helper.resourece for them